haskell/src/Monocle/Api/Config.hs

```diff
@@ -179,3 +179,31 @@ getTenantGroups Index {..} = Map.toList $ foldr go mempty (fromMaybe [] idents)
     addUser name groupName acc =
       let users = fromMaybe [] (Map.lookup groupName acc)
        in Map.insert groupName (users <> [name]) acc
+
+-- | Get the Ident name for a Given alias
+--
+-- >>> :{
+--  let
+--    index = emptyTenant "test" [createIdent "alice" ["opendev.org/Alice Doe/12345", "github.com/alice89"] []]
+--  in getIdentByAlias index "github.com/alice89"
+-- :}
+-- Just "alice"
+
+-- >>> :{
+--  let
+--    index = emptyTenant "test" [createIdent "alice" ["opendev.org/Alice Doe/12345", "github.com/alice89"] []]
+--  in getIdentByAlias index "github.com/ghost"
+-- :}
+-- Nothing
+getIdentByAlias :: Index -> Text -> Maybe Text
+getIdentByAlias Index {..} alias = getIdentByAliasFromIdents' alias =<< idents
+  where
+    getIdentByAliasFromIdents' = flip getIdentByAliasFromIdents
```

Comment:
nit: since we introduce `getIdentByAliasFromIdents`, we could arrange its parameter to avoid the flip, then `getIdentByAlias` would be:
```suggestion
getIdentByAlias Index {..} alias = getIdentByAliasFromIdents alias =<< idents
```